ee/codegen/src/context/node-context/base.ts

```diff
@@ -13,11 +13,13 @@ export declare namespace BaseNodeContext {
   interface Args<T extends WorkflowDataNode> {
     workflowContext: WorkflowContext;
     nodeData: T;
+    importOrder: readonly number[];
```

Comment:
Most important file to review.

We use a readonly array here to account for subworkflows